runtime/doc/editing.txt

```diff
@@ -1651,3 +1651,30 @@ There are three different types of searching:
    with depth limiter (/usr/**2) or upward search (;) notations.
 
  vim:tw=78:ts=8:noet:ft=help:norl:
+
+==============================================================================
+11. Trusted Files						*trust*
+
+Nvim has the ability to execute arbitrary code through the 'exrc' option. In
+order to prevent executing code from untrusted sources, Nvim has the concept of
+"trusted files". An untrusted file will not be executed without the user's
+consent, and a user can permanently mark a file as trusted or untrusted using
+the |:trust| command or the |vim.secure.read()| function.
+
+							*:trust* *E5570*
+:trust [++deny] [++remove] [{file}]
+
+			Manage files in the trust database. Without any options
+			or arguments
+			`:trust` adds the file associated with the current
+			buffer to the trust database, along with the SHA256
+			hash of its contents.
+			`++deny` marks the file associated with the current
+			buffer (or {file}, if given) as denied; no prompts will
+			be displayed to the user and the file will never be
+			executed.
+			`++remove` removes the file associated with the current
+			buffer (or {file}, if given) from the trust database.
+			Future attempts to read the file in a secure setting
```

Comment:
This is my mistake.

```suggestion
			:trust adds the file associated with the current
			buffer to the trust database, along with the SHA256
			hash of its contents.
			[++deny] marks the file associated with the current
			buffer (or {file}, if given) as denied; no prompts will
			be displayed to the user and the file will never be
			executed.
			[++remove] removes the file associated with the current
			buffer (or {file}, if given) from the trust database.
			Future attempts to read the file in a secure setting
```

Reply:
Done in 4b1d99b1cdd0ce92aa3867d0568d1c6e9a61e8df